cmd/crypto/header_test.go

```diff
@@ -96,27 +96,27 @@ var kmsParseHTTPTests = []struct {
 	{Header: http.Header{
 		"X-Amz-Server-Side-Encryption":                []string{"aws:kms"},
 		"X-Amz-Server-Side-Encryption-Aws-Kms-Key-Id": []string{"s3-007-293847485-724784"},
-		"X-Amz-Server-Side-Encryption-Context":        []string{"{}"},
+		"X-Amz-Server-Side-Encryption-Context":        []string{"e30="},
```

Comment:
```suggestion
		"X-Amz-Server-Side-Encryption-Context":        []string{base64.StdEncoding.EncodeToString("{}")},
```
... just to make it clear what we are storing.